src/storage/impl/Internal.js

```diff
@@ -30,6 +31,13 @@ class Internal {
     } else {
       this.db = new loki('medmorph.db');
     }
+    runWhenDBReady(() => {
```

Comment:
This file is supposed to be for pieces specific to the Loki data access implementation. Nothing here is loki specific so I think it should be moved to `app.js` along side the other `runWhenDBReady` functions.